src/Core/Grid/Column/Type/Common/BadgeColumn.php

```diff
@@ -54,13 +54,14 @@ protected function configureOptions(OptionsResolver $resolver)
                 'field',
             ])
             ->setDefaults([
-                'badge_type' => 'success',
+                'badge_type' => '',
```

Comment:
Hi @ks129 

I totally agree with you, but it is outside the scope of your PR and it can be considered a BC.
Instead, you can keep the current behavior and set these options here:
`src/Core/Grid/Definition/Factory/OutstandingGridDefinitionFactory.php
`
And then if you want you can create another PR.